src/frontend/src/lib/components/navigation/NavigationMenu.svelte

```diff
@@ -2,18 +2,26 @@
 	import { page } from '$app/stores';
 	import IconWallet from '$lib/components/icons/IconWallet.svelte';
 	import IconlySettings from '$lib/components/icons/iconly/IconlySettings.svelte';
+	import IconlyUfo from '$lib/components/icons/iconly/IconlyUfo.svelte';
 	import InfoMenu from '$lib/components/navigation/InfoMenu.svelte';
 	import NavigationItem from '$lib/components/navigation/NavigationItem.svelte';
 	import { networkId } from '$lib/derived/network.derived';
 	import { i18n } from '$lib/stores/i18n.store';
-	import { isRouteSettings, isRouteTransactions, networkParam } from '$lib/utils/nav.utils.js';
+	import {
+		isRouteDappExplorer,
+		isRouteSettings,
+		isRouteTransactions,
+		networkParam
+	} from '$lib/utils/nav.utils.js';
```

Comment:
unrelated to this PR: @AntonioVentilii-DFINITY but, it looks like your editor is adding some `.js` at the end of the imports.